src/ctapipe/image/extractor.py

```diff
@@ -1308,7 +1308,7 @@ def __call__(
         self, waveforms, tel_id, selected_gain_channel, broken_pixels
     ) -> DL1CameraContainer:
         container = DL1CameraContainer(
-            image=np.nanvar(waveforms, dtype="float32", axis=2),
+            image=np.nanvar(waveforms, dtype="float32", keepdims=False, axis=2),
```

Comment:
A bit weird that you need to set `keepdims=False` explicitly, but ok.

Reply:
i will check again if i make it work without that

Reply:
Should work without it: 
```py3
x  = np.random.uniform(size=(2,1800, 1000))

In [1]: np.nanvar(x, axis=2).shape
Out[2]: (2, 1800)

In [2]: np.nanvar(x, axis=2, keepdims=True).shape
Out[2]: (2, 1800, 1)

In [3]: np.nanvar(x, axis=2, keepdims=False).shape
Out[3]: (2, 1800)
```